spec/fixtures/files/crime_apply_data/closed_applications.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "5aa4c689-6fb5-47ff-9567-5efe7f8ac211",
+  "reference": "5230234344",
+  "schema_version": "1.0",
+  "status": "returned",
+  "created_at": "2022-11-14T16:56:51.000+00:00",
+  "date_stamp": "2022-10-21T16:19:00.000+00:00",
+  "submitted_at": "2022-11-14T16:58:15.000+00:00",
+  "provider_details": [],
+  "client_details": {
+    "applicant": {
+      "home_address": {
+        "address_line_one": "1",
+        "address_line_two": "",
+        "city": "London",
+        "country": "London",
+        "postcode": "SW1A 2AA"
+      },
+      "first_name": "Ella",
+      "last_name": "FITZGERALD",
+      "full_name": "Ella FITZGERALD",
+      "nino": "NC123458A",
+      "date_of_birth": "1980-06-01",
+      "telephone_number": "07612345649",
+      "correspondence_address_type": "home_address"
+    }
+  },
+  "interests_of_justice": [
+    {
+      "reason": "Justification provided by provider.",
+      "type": "loss_of_liberty"
+    }
+  ],
+  "case_details": {
+    "case_type": "summary_only",
+    "codefendants": [],
+    "offences": [
+      {
+        "class": "Class C or B",
+        "dates": ["2020-12-12"],
+        "name": "Robbery"
+      }
+    ],
+    "hearing_court_name": "Caernarfon Crown Court",
+    "hearing_date": "2023-12-12",
+    "urn": "12AB3456789"
+  },
+  "application_type": "initial_application"
+}
```

Comment:
I think there may be a config issue on your IDE - it should add a new line at the end of files - pretty simple fix. This can cause issues on unix based OSs